<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="cbCategory">
        <property name="whatsThis">
         <string extracomment="Select sort order or category"/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbDate"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbAmount"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbCheckNum"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbDescription"/>
      </item>
      <item>
       <widget class="QLabel" name="labelSearchIn">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search In:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbSearchIn"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Group By:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbGroupBy"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listEntries">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::Batched</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCalendarWidget" name="calendar2"/>
    </item>
    <item>
     <widget class="QCalendarWidget" name="calendar1"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnReadFile">
          <property name="text">
           <string>Read Check File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnChart">
          <property name="text">
           <string>Predicted</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMngPredict">
          <property name="text">
           <string>Manage Predictions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBackup">
          <property name="text">
           <string>Backup DB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCleanDB">
          <property name="text">
           <string>Clean DB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnListToFile">
          <property name="text">
           <string>List To File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>388</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>listEntries</zorder>
   <zorder>btnChart</zorder>
   <zorder>btnReadFile</zorder>
   <zorder>cbCategory</zorder>
   <zorder>cbDate</zorder>
   <zorder>cbAmount</zorder>
   <zorder>cbCheckNum</zorder>
   <zorder>cbDescription</zorder>
   <zorder>calendar1</zorder>
   <zorder>cbSearchIn</zorder>
   <zorder>labelSearchIn</zorder>
   <zorder>label</zorder>
   <zorder>cbGroupBy</zorder>
   <zorder>calendar2</zorder>
   <zorder>btnMngPredict</zorder>
   <zorder>btnBackup</zorder>
   <zorder>btnCleanDB</zorder>
   <zorder>btnListToFile</zorder>
   <zorder>horizontalSpacer</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
